src/quo/components/wallet/token_input/view.cljs

```diff
@@ -92,36 +84,49 @@
                                         (oops/oget "nativeEvent.selection")
                                         (js->clj :keywordize-keys true)
                                         (on-selection-change))))]
-    (fn [{:keys [token customization-color show-keyboard? crypto? currency value error? selection]
+    (fn [{:keys [token customization-color show-keyboard? crypto? currency value error? selection
+                 handle-on-swap]
           :or   {show-keyboard? true}}]
```

Comment:
it can be quite hard to distinguish what changes you have actually made in code blobs like this. 
It would be nice if you could leave some comments to help find the changes :)

Reply:
The main reason is this one https://github.com/status-im/status-mobile/pull/20000/files#r1601228153